src/index.html

```diff
@@ -9,6 +9,7 @@
   <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet"> 
   <link href="https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@300;400;500;700;900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
double-check if we need this

Reply:
Can you remind me what the alternative is for this?

Reply:
importing in material alone